File: src/net/eduard/essentials/command/ListCommand.java

package net.eduard.essentials.command;

import org.bukkit.command.Command;
import org.bukkit.command.CommandSender;

import net.eduard.api.lib.modules.Mine;
import net.eduard.api.lib.manager.CommandManager;

public class ListCommand extends CommandManager {

	public String message = "§aTem $players jogadores online!";
	public ListCommand() {
		super("list");
	}
	@Override
	public boolean onCommand(CommandSender sender, Command command,
			String label, String[] args) {
		sender.sendMessage( message.replace("$players", ""+Mine.getPlayers().size()));
		return true;
	}
}

File: src/main/java/net/eduard/essentials/command/ListCommand.kt
package net.eduard.essentials.command

import net.eduard.api.lib.manager.CommandManager
import net.eduard.api.lib.modules.Mine
import org.bukkit.command.Command
import org.bukkit.command.CommandSender

class ListCommand : CommandManager("list", "players", "jogadores", "onlines") {
    var message = "§aTem %players jogadores online!"
    override fun command(sender: CommandSender, args: Array<String>) {
        sender.sendMessage(message.replace("%players", "" + Mine.getPlayers().size))
    }
}